test/itkInMemoryMontageTestHelper.hxx

```diff
@@ -28,692 +28,557 @@
-inMemoryMontageTest( const itk::TileLayout2D& stageTiles, const std::string& inputPath, std::string& outFilename,
-                     TestVariation variation, unsigned streamSubdivisions)
-{
-  outFilename = outFilename + std::to_string( static_cast<unsigned int>(variation) ) + ".mha";
-
-  if (variation == TestVariation::UOrigin_USpacing_UTransform)
-  {
-    return executeInMemoryMontageTest0<PixelType,AccumulatePixelType>(stageTiles, inputPath,
-                                                                      outFilename, streamSubdivisions);
-  }
-  else if (variation == TestVariation::UOrigin_DSpacing_UTransform)
-  {
-    return executeInMemoryMontageTest1<PixelType,AccumulatePixelType>(stageTiles, inputPath,
-                                                                      outFilename, streamSubdivisions);
-  }
-  else if (variation == TestVariation::UOrigin_USpacing_DTransform)
-  {
-    return executeInMemoryMontageTest2<PixelType,AccumulatePixelType>(stageTiles, inputPath,
-                                                                      outFilename, streamSubdivisions);
-  }
-  else if (variation == TestVariation::UOrigin_DSpacing_DTransform)
-  {
-    return executeInMemoryMontageTest3<PixelType,AccumulatePixelType>(stageTiles, inputPath,
-                                                                      outFilename, streamSubdivisions);
-  }
-  else if (variation == TestVariation::DOrigin_USpacing_UTransform)
-  {
-    return executeInMemoryMontageTest4<PixelType,AccumulatePixelType>(stageTiles, inputPath,
-                                                                      outFilename, streamSubdivisions);
-  }
-  else if (variation == TestVariation::DOrigin_DSpacing_UTransform)
-  {
-    return executeInMemoryMontageTest5<PixelType,AccumulatePixelType>(stageTiles, inputPath,
-                                                                      outFilename, streamSubdivisions);
-  }
-  else if (variation == TestVariation::DOrigin_USpacing_DTransform)
-  {
-    return executeInMemoryMontageTest6<PixelType,AccumulatePixelType>(stageTiles, inputPath,
-                                                                      outFilename, streamSubdivisions);
-  }
-  else if (variation == TestVariation::DOrigin_DSpacing_DTransform)
-  {
-    return executeInMemoryMontageTest7<PixelType,AccumulatePixelType>(stageTiles, inputPath,
-                                                                      outFilename, streamSubdivisions);
-  }
-  else
-  {
-    return EXIT_FAILURE;
-  }
-}
+      for (itk::TileRow2D tileRow : stageTiles)
+      {
+        TransformPtrRow transform_row;
+        for (itk::Tile2D tile : tileRow)
+        {
+          typename MontageType::TransformPointer transform = MontageType::TransformType::New();
+
+          auto offset = transform->GetOffset();
+          for (unsigned i = 0; i < MontageType::TransformType::SpaceDimension; i++)
+          {
+            offset[i] = tile.Position[i];
```

Comment:
This might need a minus sign: `offset[i] = -tile.Position[i];`

Reply:
That function simply takes whatever positions are in the stageTiles variable and sets them into a Transform2D structure.  I use that function in places where I want the origins from the file to be used as transforms.

Reply:
I think that transforms do not map from source to target, but from target to source (the opposite of intuitive). This is because the usual case for transforms is resampling, and you want to evaluate it at pixels of the target without having to invert it.

Reply:
So you're saying that a negative number in the X coordinate of a transform would move the tile in the positive X direction?  And a negative number in the Y coordinate will move the tile in the positive Y direction?

Reply:
Yes. But this might stay like this until I get around to fixing it.

When you think this is ready to merge, remove WIP prefix from title.

Reply:
If this PR is otherwise ready, no. If you will do other additional work, you could explore minus sign here and maybe in a few other places.

Reply:
OK.  This PR is ready to be merged in, so I will remove the WIP label.